src/main/java/entities/SuggestedTitleChecker.java

```diff
@@ -0,0 +1,15 @@
+package entities;
+
+/**
+ * Abstract interface to check whether a suggested title is a valid title for a story.
+ */
+public interface SuggestedTitleChecker {
+
+    /**
+     * This abstract method checks whether the given title is a valid story title. The specific criteria for
+     * determining whether a title is valid depends on the implementation of the SuggestedTitleChecker interface.
+     * @param title the title which we want to check for validity
```

Comment:
Somewhere in this method doc, mention that there is a "precondition" that the inputted title MUST have trimmed spaces AND all consecutive interior spaces are cut down to one. This way, anyone implementing this interface can expect these conditions to be true and anyone calling an implementation of this interface knows they must first trim and cut down spaces in their string

Reply:
done.